test/functional/api/cas/casadm.py

```diff
@@ -23,7 +25,13 @@ def help(shortcut: bool = False):
 
 def start_cache(cache_dev: Device, cache_mode: CacheMode = None,
                 cache_line_size: CacheLineSize = None, cache_id: int = None,
-                force: bool = False, load: bool = False, shortcut: bool = False):
+                force: bool = False, load: bool = False, shortcut: bool = False,
+                kernel_params: KernelParameters = None):
+    if kernel_params is None:
+        kernel_params = KernelParameters()
```

Comment:
Why not pass new blank instance of `KernelParameters` instead of None by default? It would be initialized anyway.

Reply:
Makes sense :D